codespell_lib/data/dictionary.txt

```diff
@@ -8933,8 +8945,13 @@ halp->help
 halpoints->halfpoints
 hammmer->hammer
 hampster->hamster
+handel->handle
```

Comment:
The composer? https://en.wikipedia.org/wiki/George_Frideric_Handel

Reply:
we're not doing names AFAIR, so I think this is valid

Reply:
Okay fair enough.